packages/jest-matcher-utils/src/index.js

```diff
@@ -75,6 +90,9 @@ const stringify = (obj: any): string => {
   });
 };
 
+// highlight an object
+const h = (obj: any) => chalk.cyan.bold(stringify(obj));
```

Comment:
Can we call this highlight? Then we don't need the comment and people will understand what `h` means.